<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>assetCheckUI</class>
 <widget class="QWidget" name="assetCheckUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>1060</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asset Checker V1.0</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QTreeWidget" name="checkTree">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="ExportBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{ 
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:1 #116776, stop:0 #116776);
padding: 5px;
border-radius: 2px;
}

QPushButton:hover
{
background-color: #1d8ba3;
border-radius: 2px;
}

QPushButton:pressed
{
background-color: #1d1d1d;
border-radius: 2px;
}

QPushButton:disabled
{
background-color: #4b4b4b;
border-radius: 2px;
}</string>
     </property>
     <property name="text">
      <string>Publish model</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="fixBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Run Auto Fix</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="checkBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Run Checks</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="checkTypeBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{ 
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:1 #116776, stop:0 #116776);
padding: 5px;
border-radius: 2px;
}

QPushButton:hover
{
background-color: #1d8ba3;
border-radius: 2px;
}

QPushButton:pressed
{
background-color: #1d1d1d;
border-radius: 2px;
}

QPushButton:disabled
{
background-color: #4b4b4b;
border-radius: 2px;
}</string>
     </property>
     <property name="text">
      <string>model</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
